server.Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM node:12.14.0-alpine
```

Comment:
As this is only for the server, I think the Dockerfile should be in that folder.
Besides, we should add this image to docker-compose file there to orchestrate it with PostgreSql image for db.

Reply:
For ease of k8s deployments, we decided to leave it here.